src/gConsent/guard/isGConsentAttributes.ts

```diff
@@ -34,7 +34,7 @@ function isResourceAccessModeArray(x: unknown): x is Array<ResourceAccessMode> {
 }
 
 function isGConsentStatus(x: unknown): x is GConsentStatus {
-  return typeof x === "string" && (CONSENT_STATUS as Set<string>).has(x);
+  return typeof x === "string" && (ACCESS_STATUS as Set<string>).has(x);
```

Comment:
```suggestion
  return typeof x === "string" && ACCESS_STATUS.has(x);
```

`ACCESS_STATUS` should already be a `Set<string>` per its definition above?

Reply:
Is this just because we're merging two sets? What if we define the set members internally as two arrays, and then export 3 sets? Would that fix?

Reply:
What about https://github.com/inrupt/solid-client-access-grants-js/pull/461/commits/1bfe0f8a1b47824297aea5f332afc0b0d0359216 ? It also makes the sets read-only.

Reply:
I like it, but I don't understand it.

Reply:
The main difference boils down to
```
const stringArray = ["a", "b", "c"] // inferred to string[]
const narrowArray = ["a", "b", "c"] as const // inferred to  readonly ["a", "b", "c"]
```
The sets being generic, they just reflect the type of the collection they wrap around. The change in https://github.com/inrupt/solid-client-access-grants-js/commit/1bfe0f8a1b47824297aea5f332afc0b0d0359216 moves from the latter to the former.

Reply:
Okay, cool!